guides/common/modules/con_troubleshooting-postgresql.adoc

```diff
@@ -0,0 +1,30 @@
+[id="Troubleshooting_Postgresql_{context}"]
+= Troubleshooting PostgreSQL
+
+If the PostgreSQL 10 module has already been enabled, perform a module reset:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module reset postgresql
+----
+
+If a database was previously created using PostgreSQL 10, perform an upgrade:
```

Comment:
Should we go ahead and update this to `12`?

Reply:
Updated :+1: